lib/utils.py

```diff
@@ -0,0 +1,32 @@
+from controller import helpers
```

Comment:
I see we have a ./lib/controller/helpers.py that isn't necessarily related to controllers.  Could you merge that into this new utils.py?